metapool/metapool.py

```diff
@@ -96,6 +96,9 @@ def read_pico_csv(f, sep='\t', plate_reader='Synergy_HT',
     pico_df[conc_col_name] = \
         pd.to_numeric(pico_df[conc_col_name], errors='coerce')
 
+    # limit concentration range (0 - 60 )
+    pico_df[conc_col_name] = np.clip(pico_df[conc_col_name], 0, 60)
```

Comment:
I thought this should only be done for `SpectraMax_i3x` if so can you add something like: `if plate_reader == 'SpectraMax_i3x': ....`